proto/cosmos/app/v1alpha1/module.proto

```diff
@@ -0,0 +1,93 @@
+syntax = "proto3";
+
+package cosmos.app.v1alpha1;
+
+import "google/protobuf/descriptor.proto";
+
+extend google.protobuf.MessageOptions {
+  // is_module indicates that this proto type is a config object for an app module
+  // and optionally provides other descriptive information about the module.
+  // It is recommended that a new module config object and go module is versioned
+  // for every state machine breaking version of a module. The recommended
+  // pattern for doing this is to put module config objects in a separate proto
+  // package from the API they expose. Ex: the cosmos.group.v1 API would be
+  // exposed by module configs cosmos.group.module.v1, cosmos.group.module.v2, etc.
+  ModuleDescriptor is_module = 57193479;
+}
+
+// ModuleDescriptor describes an app module.
+message ModuleDescriptor {
+  // go_import names the package that should be imported by an app to load the
+  // module in the runtime module registry. Either go_import must be defined here
+  // or the go_package option must be defined at the file level to indicate
+  // to users where to location the module implementation. go_import takes
+  // precedence over go_package when both are defined.
+  string go_import = 1;
+
+  // use_package refers to a protobuf package that this module
+  // uses and exposes to the world. In an app, only one module should "use"
+  // or own a single protobuf package. It is assumed that the module uses
+  // all of the .proto files in a single package.
+  repeated PackageReference use_package = 2;
```

Comment:
What's the reason for adding this field?

Reply:
Because modules need to pin their file descriptors and also "own" certain resources Msg and query servers or orm definitions